starting-files/ConnectorConfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "ServerConfig":{
+    "URL":"https://192.168.0.124/piwebapi/omf",
+    "WebID":"s0U1IjG6kMOEW7mxyHCuX2mAUEktU0VSVkVSLVBD",
+    "Credentials":"UEktU2VydmVyOk1hbmNoZXN0ZXIxMjMh",

```

Comment:
This information uses example credentials right?

Reply:
@oliver-walker-hms  
Do not have credentials in tracked files. Examples for customers should use an obvious string that should be replaced.